gravitybridge/assetlist.json

```diff
@@ -296,6 +296,51 @@
         }
       ]
     },
+    {
+      "description": "Gravity Bridge sDAI",
+      "denom_units": [
+        {
+          "denom": "gravity0x83F20F44975D03b1b09e64809B757c47f942BEeA",
+          "exponent": 0
+        },
+        {
+          "denom": "gsdai",
+          "exponent": 18
+        }
+      ],
+      "base": "gravity0x83F20F44975D03b1b09e64809B757c47f942BEeA",
+      "name": "Savings Dai",
+      "display": "gsdai",
+      "symbol": "sDAI",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/sdai.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/sdai.svg"
+      },
+      "traces": [
+        {
+          "type": "synthetic",
+          "counterparty": {
```

Comment:
Since there might be traces in between, I think it's probably safest to remove this trace. A full trace can be compiled by external software, but adding an incomplete trace path here may interfere with that since it's ambiguous which origin path to follow.

Reply:
Fix this myself